examples/data-channels-close/main.go

```diff
@@ -90,10 +90,13 @@ func main() {
 	err = peerConnection.SetRemoteDescription(offer)
 	util.Check(err)
 
-	// Sets the LocalDescription, and starts our UDP listeners
+	// Create answer
 	answer, err := peerConnection.CreateAnswer(nil)
 	util.Check(err)
 
+	// Sets the LocalDescription, and starts our UDP listeners
+	util.Check(peerConnection.SetLocalDescription(answer))
```

Comment:
Maybe put the Check on a new line? It just shows off the API a tiny bit better, I feel?

Sidenote: I've seen util.Check() (and other parts of util) leak into peoples projects. We should think about ways to avoid that.

Reply:
It would be a little silly, but maybe move util under examples/internal/util? Just to make it a bit more explicit... Maybe also drop a README.md/and or code comments to the effect that nothing in that package is intended to be public API and could break at any moment.

Reply:
Or maybe remove util all together? I think it's an anti-pattern.

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
👍 to moving the util stuff under an `internal` (and just not allowing people to use it as a library)

I see a lot of people using and it was never really intended, we eventually will break it also.